<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>645</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>961</width>
    <height>645</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>961</width>
    <height>645</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>338</y>
      <width>103</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'monospace'; font-size:18pt; font-weight:600; font-style:italic;&quot;&gt;Viewed:&lt;/span&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>385</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; font-style:italic;&quot;&gt;To watch:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetViewed">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>377</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgeWatch">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>49</y>
      <width>531</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNewViewed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>378</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNewWatch">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>49</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonViewedTransfer">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteWatch">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>49</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteViewed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>378</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>88</y>
      <width>373</width>
      <height>473</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="lineEditNameFilm">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Name film</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxGenre">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Film Genre</string>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Film Genre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Action</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Adventure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comedy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Drama</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Crime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Horror</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fantasy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Romance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thriller</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Animation</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_genre">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Age">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAge">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="maxVisibleItems">
        <number>10</number>
       </property>
       <item>
        <property name="text">
         <string>Age rating of the film</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18+</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCountry">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Сountry</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Release date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditRelease">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Rating:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRating">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderRating">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="sliderPosition">
          <number>6</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBothSides</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;10&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditOpinion">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Opinion about the film...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEdit">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAccept">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Аccept</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>10</y>
      <width>371</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; font-style:italic;&quot;&gt;Information about the film&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMode">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>52</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mode: expectation</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Viewed Statistics</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Statistics pending viewing</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>General Statistics</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>QuickStart</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>About the author</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
